src/Microsoft.IdentityModel.Logging/LogHelper.cs

```diff
@@ -401,7 +401,7 @@ private static string RemovePII(object arg)
 
         internal static bool IsCustomException(Exception ex)
         {
-            return ex.GetType().FullName.StartsWith("Microsoft.IdentityModel.", StringComparison.Ordinal);
+            return ex.GetType().FullName!.StartsWith("Microsoft.IdentityModel.", StringComparison.Ordinal);
```

Comment:
why is this needed?

Reply:
IDE throws error code CS8602-Dereference of a possibly null reference. FullName is not nullable in System so it causes noise